### ISSUE TITLE
data-prev(ious)-button-aria-label

### ISSUE THREAD

**Issue by Malvoz** [ampproject/docs#1356]
You can see both the common attribute `data-prev-button-aria-label` and `data-previous-button-aria-label` mentioned in https://www.ampproject.org/docs/reference/components/amp-carousel#data-prev-button-aria-label-(optional) (see the example lower on the page) and https://github.com/ampproject/amphtml/pull/9431/files

Currently,  `data-prev-button-aria-label` works, `data-previous-button-aria-label` does not.

**Comment by matthiasrohmer** [comment]
Thanks for your report, @Malvoz! I am not quite sure if i got you correctly: you mean `data-previous-button-aria-label` doesn't work on your site? Then this would be an issue in `amphtml` instead of `docs`. Can you please clarify? 🙂

**Comment by matthiasrohmer** [comment]
Closing hence there was no reaction.
